tasks/utils.py

```diff
@@ -552,3 +552,14 @@ def timed(name=""):
     finally:
         end = time.time()
         print(f"{name} completed in {end-start:.2f}s")
+
+
+class stop_watch:
```

Comment:
Why not add an extra param to `timed()` instead of creating a whole new method?

Eg signature:
```python
timed(name="", quiet=False):
    ....
```

Reply:
Good question, and I have no good reason :sweat_smile: 
I updated the PR with the proposed change